engines/rbd.c

```diff
@@ -127,6 +135,7 @@ static int _fio_setup_rbd_data(struct thread_data *td,
 
 }
 
+#define EVENT_SOCKET_TYPE_EVENTFD 2 /* add for rbd poll */
```

Comment:
Why do we have to define this? If the library is new enough to have support for image notification, doesn't it also include this define in the header?